tests/test-general.php

```diff
@@ -246,7 +262,6 @@ public function inaccessible() {
 			[ '/category/archives/', 'is_category' ], // Tax archive.
 			[ '/author/test_author/', 'is_author' ], // Author archive.
 			[ '/?s=Lorem+ipsum', 'is_search' ], // Search.
-			[ '/rss/', 'is_feed' ], // Feeds.
```

Comment:
why axe the feed?

Reply:
I'm going to merge this to my branch to unblock getting the fix in place, but I'm curious about this as well. Can we bring it back?